OJP_JourneySupport.xsd

```diff
@@ -403,30 +403,30 @@
 			<xs:choice>
 				<xs:element name="PlaceRef" type="PlaceRefStructure">
 					<xs:annotation>
-						<xs:documentation>Spatial location.</xs:documentation>
+						<xs:documentation>Static place.</xs:documentation>
 					</xs:annotation>
 				</xs:element>
-				<xs:element name="TripLocation" type="TripLocationStructure">
+				<xs:element name="TripPlace" type="TripPlaceStructure">
 					<xs:annotation>
-						<xs:documentation>Location within a (moving) vehicle.</xs:documentation>
+						<xs:documentation>Place within a (moving) vehicle.</xs:documentation>
```

Comment:
I really wonder if this is the wording used in Transmodel.

Reply:
In TRANSMODEL surely the term place is not used for what we call a TripPlace here. In lack of another term I simply used TripPlace. Perhaps we should request a term for this kind of "logical position within a vehicle" from TRANSMODEL. However, in the meantime TripLocation seems to be more wrong than TripPlace.

Reply:
Change the documentation part to something that does not suggest "seating".

Reply:
Please check again the new description.